docs/Project-Configuration/Project-Configuration-Files.md

```diff
@@ -1535,7 +1535,8 @@ builds.
 To address this, project files can define a top-level `ignore`
 list of gitignore-style globs which tell Evergreen to not automatically
 run tasks for commits that only change ignored files, and we will not 
-create PR patches but instead send a successful status. 
+create PR patches but instead send a successful status for all required
```

Comment:
can we say "and the Evergreen check" since that one we'll send even if it's not required?